pegasus/sites.v3/code.org/public/css/student.css

```diff
@@ -1,55 +1,24 @@
- .full-resource-block {
-   width: 100%;
-   height: 244px;
-   margin-top: 10px;
- }
+.full-resource-block {
+  display: flex;
+  flex-direction: column;
+}
 
 .description {
-   padding-bottom: 10px;
-   color: #fff;
- }
-
- .resource-title {
```

Comment:
Is this css used anywhere else on the site?

Reply:
no, i looked for all other files in pegasus where we import `student.css`, and this is the only place

Reply:
maybe pegasus/sites.v3/hourofcode.com/public/beyond.haml?

Reply:
and is pegasus/sites.v3/hourofcode.com/public/css/beyond.css duplicate css that should reflect the changes here?

Reply:
`beyond.haml` [imports `beyond.css`](https://github.com/code-dot-org/code-dot-org/blob/92fadfce547d2592516abd811dd399287fc352ba/pegasus/sites.v3/hourofcode.com/public/beyond.haml#L6), so the styles there are separate from the styles in `student.css` (just named similarly)